editing/fx/passes/eps.py

```diff
@@ -389,6 +392,14 @@ def run_pass(self, gm : fx.GraphModule):
                             print("[AnnotateEpsPass] Mismatching input epsilons in node with no eps propagation function! Eps propagation will likely be wrong!")
                             print(f"                    -> Node: {node.name}, Key: {k}, eps_in: {all_eps}")
                             if (self.verbose): print(f"[AnnotateEpsPass] Using identity epsilon propagation on node with op {node.op}, target {node.target}!")
+
+                        if node.op == "call_function" and node.target == torch.cat:
```

Comment:
Please extract this into a more proper format - maybe a function that returns all_eps at the top of this code block rather than hacking this in here.

Reply:
Also, why do you check the `eps_in` of the arguments and not the `eps_out`?

Reply:
I agree with Philip; this is strange... @Scheremo is this on purpose?

Reply:
Should probably be eps_out!

Reply:
@Scheremo: Done [here](https://github.com/pulp-platform/quantlib/pull/15/commits/09d2983fd33738ad3b193fa3917588587c075cad)

@Xeratec: Done [here](https://github.com/pulp-platform/quantlib/pull/15/commits/ed0d20c296124c28c4be0d9e9f3b2e0c5a5f63cf)